graf2d/win32gdk/inc/TGWin32VirtualXProxy.h

```diff
@@ -66,7 +66,7 @@ friend class TGWin32;
    Float_t   GetTextMagnitude() override;
    Window_t  GetWindowID(Int_t wid) override;
    Bool_t    HasTTFonts() const override;
-   Int_t     InitWindow(ULong_t window) override;
+   Int_t     InitWindow(ULongPtr_t window) override;
```

Comment:
```suggestion
   Int_t     InitWindow(ULongptr_t window) override;
```
Same for the other occurrences